force-app/main/default/classes/AccountTriggerHandler.cls

```diff
@@ -0,0 +1,72 @@
+public with sharing class AccountTriggerHandler {
+    public static void setTypeProspectAfterInsert(List<Account> newAccounts) {
+        List<Account> accountsToUpdate = new List<Account>();
+        //System.debug('number of new accounts: ' + newAccounts.size());
+        for (Account acc : newAccounts) {
+            if (acc.Type == null) {
+                Account accToUpdateAfterInsert = new Account(Id = acc.Id , Type = 'Prospect');
+                accountsToUpdate.add(accToUpdateAfterInsert);
+                //System.debug('Added to update list: ' + accToUpdateAfterInsert);
+            }
+        }
+        //System.debug('number of accounts being updated with prospect: ' + accountsToUpdate.size());
+
+        if (!accountsToUpdate.isEmpty()) {
+            update as system accountsToUpdate;
+        }
+    }
+
+    public static void addressCopyAfterInsert(List<Account> newAccounts) {
```

Comment:
Double check the context here too. Remember, after insert means the Trigger.new has already been saved